PRFCS/prfc-2.md

```diff
@@ -62,8 +68,6 @@ struct Token {
 ## Required Views 
```

Comment:
In Summary section line 14, we should include "Required Views" too? Or we can follow how we do it in PRFC1, saying `"The below section lists the set of methods that all smart contracts that want to be PRFC 1-compliant must implement....."`